dell/xps/15-9560/default.nix

```
@@ -4,36 +4,22 @@
     ../../../common/cpu/intel/kaby-lake
     ../../../common/pc/laptop
     ./xps-common.nix
-
-    # FIXME: remove this when bumblebee works again
-    ../../../common/gpu/nvidia/disable.nix
+    ../../../common/gpu/nvidia
   ];
 
+  hardware.graphics = {
+    enable = true;
+    driSupport = true;
+    driSupport32Bit = true;
+  };
```

driSupport is no longer and 32-bit drivers is probably a bit obsolete outside of older games:

```suggestion
  hardware.graphics.enable = true;
```